examples/e-commerce/src/app.mobile.css

```diff
@@ -0,0 +1,274 @@
+    background: #fff;
+    border-radius: 16px;
+    left: 0;
+    max-width: initial;
+    padding-bottom: 4rem;
+    position: fixed;
+    top: 0;
+    transform: translateY(120vh);
+    transition: transform 300ms cubic-bezier(0.465, 0.183, 0.153, 0.946);
+    width: 100%;
+    will-change: transform;
+    z-index: 1;
+  }
+
+  .container-filters .container-header,
+  .container-filters .container-body {
+    padding: 2rem 2rem 0 2rem;
+  }
+
+  .filtering .header {
+    /*
+      Closing panel on outter click didn't work on mobile safari.
+      This is one of the workarounds from the following: 
+      https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event#Safari_Mobile 
+    */
+    cursor: pointer;
+  }
+
+  .filtering .header-logo {
+    left: 50%;
+    pointer-events: none;
+    position: absolute;
+    top: 1.5rem;
+    transform: translateX(-50%);
+  }
+
+  .filtering .header-title,
+  .filtering .container-results,
+  .container-filters-footer {
+    display: none;
+  }
+
+  .filtering .container-filters {
+    position: absolute;
+    transform: translateY(4rem);
+  }
+
+  .filtering .container-filters-footer {
+    background-color: #fff;
+    border-top: 1px solid #ebecf3;
+    bottom: 0;
+    display: flex;
+    justify-content: space-between;
+    margin: 0 -2rem;
+    padding: 1rem;
+    position: fixed;
+    width: 100%;
+    z-index: 5; /* to avoid collision with noUiSlider */
+  }
+
+  .container-filters-footer-button-wrapper {
+    width: calc(50% - 0.5rem);
+  }
+
+  .ais-ClearRefinements-button,
+  .container-filters-footer .button {
+    background-color: rgba(65, 66, 71, 0.08);
+    border: none;
+    border-radius: 8px;
+    cursor: pointer;
+    display: flex;
+    font: inherit;
+    font-size: 0.75rem;
+    font-weight: bold;
+    justify-content: center;
+    padding: 12px;
+    text-align: center;
+    width: 100%;
+  }
+
+  .container-filters-footer .button-primary {
+    background-color: #e2a400;
+    color: #fff;
+  }
+
+  /* Filters button that triggers the overlay */
+
+  .filtering .filters-button {
+    display: none;
+  }
+
+  .filters-button {
+    align-items: center;
+    background-color: #e2a400;
+    border: none;
+    border-radius: 8px;
+    bottom: 2rem;
+    box-shadow: 0 4px 22px 0 rgba(185, 135, 0, 0.5);
+    color: #fff;
+    cursor: pointer;
+    display: flex;
+    font: inherit;
+    font-size: 0.875rem;
+    font-weight: bold;
+    justify-content: center;
+    left: 50%;
+    min-height: 40px;
+    min-width: 112px;
+    position: fixed;
+    transform: translateX(-50%);
+  }
+
+  .filters-button svg {
+    height: 14px;
+    margin-right: 8px;
+    width: 16px;
+  }
+
+  .container {
+    padding: 4rem 2rem;
+  }
+
+  .container-options {
+    display: none;
+  }
+
+  .header {
+    background: linear-gradient(to bottom, #ae8600, #885b01);
+    background-image: url('./images/cover-mobile.png');
+    background-image: url('./images/cover-mobile.png'),
```

Comment:
Even though this is for mobile it's still quite big (+800 KB).
It could be worth converting to JPEG (this format is often more performant for picture that aren't simple illustrations, like in this case).

Also maybe the dimensions can be smaller for mobile.

Reply:
That's true – I thought about following "[The CSS background-image property as an anti-pattern](https://nystudio107.com/blog/the-css-background-image-property-as-an-anti-pattern)" in another pass when we'll deal with accessibility.